features/user_can_navigate_to_a_specific_restaurant_page.feature

```diff
@@ -9,8 +9,10 @@ Feature: Visitor can view details about a specific restaurant
     | ThaiTanic       | Thailands finest food, watch out for the iceberg |
     | Fu King Chinese | Best chinese food in town                        |
```

Comment:
This restaurant is not being used for testing, we can remove it or better test a second restaurant. :)

Reply:
We'll remove it